templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,40 @@ contents:
       echo ""
     }
 
+    get_interface_from_ip_hint() {
+      local ip_hint=""
+      local ip_hint_file="$1"
+      local extra_bridge="$2"
+      local iface=""
+
+      if [ ! -f "${ip_hint_file}" ]; then
+        return
+      fi
+
+      ip_hint=$(cat ${ip_hint_file})
+      if [ "${ip_hint}" != "" ]; then
```

Comment:
we could use more golang style avoiding extra indents by returning on failure here.

```sh
if [[ -z "${ip_hint}"  ]]; then
    return
fi
if [ "${extra_bridge}" != "" ]; then
...
```

Reply:
err, well I don't know why we are not using `-n` and `-z` everywhere when checking for empty strings.

```sh
if [[ -z "${ip_hint}"  ]]; then
    return
fi
if [[ -n "${extra_bridge}" ]]; then
...